WakaTime/WakaTimeConstants.cs

```diff
@@ -1,12 +1,44 @@
-﻿namespace WakaTime
+﻿using System;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace WakaTime
 {
     internal static class WakaTimeConstants
-    {
-        internal const string CurrentWakaTimeCliVersion = "4.1.0"; // https://github.com/wakatime/wakatime/blob/master/HISTORY.rst
+    {        
         internal const string CliUrl = "https://github.com/wakatime/wakatime/archive/master.zip";
         internal const string NativeName = "WakaTime";
         internal const string PluginName = "notepadpp-wakatime";
         internal const string EditorName = "notepadpp";
         internal const string CliFolder = @"wakatime-master\wakatime\cli.py";
+        internal static string UserConfigDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
+
+        internal static Func<string> PluginDir = () =>
+        {
+            var pluginConfigDir = new StringBuilder(Win32.MAX_PATH);
+            Win32.SendMessage(PluginBase.NppData._nppHandle, NppMsg.NPPM_GETPLUGINSCONFIGDIR, Win32.MAX_PATH, pluginConfigDir);
+            return pluginConfigDir.ToString();
+        };
+
+        internal static Func<string> CurrentWakaTimeCliVersion = () =>
+        {
+            var regex = new Regex(@"(__version_info__ = )(\(( ?\'[0-9]\'\,?){3}\))");
+            var client = new WebClient();
+            var about = client.DownloadString("https://raw.githubusercontent.com/wakatime/wakatime/master/wakatime/__about__.py");
```

Comment:
WebClient.DownloadString can cause an exception (https://msdn.microsoft.com/en-us/library/fhd1f0sw(v=vs.110).aspx) so let's wrap this in a try/catch block and return `string.Empty` if it fails.